build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -32,7 +32,25 @@
         <Link>app.config</Link>
       </None>
     </ItemGroup>
+  </Target>
+
+  <!-- 
+    In order to leverage LUT testing we need to have both of the follownig packages in 
+    our build.  
+
+        - xunit.runner.desktop
+        - xunit
 
+    Both of these include xunit.abstractions.dll in different forms: content and reference
+    respectively.  This creates a double write violation in our build.  The content item
+    is unnecessary for us since we reference the xunit package hence remove it here. 
```

Comment:
Please link to the tracking bug for this issue on either the xunit or NuGet side, which ever the current belief is more to blame.